plugins/module_utils/net_map/networking_definition.py

```diff
@@ -1377,6 +1557,68 @@ def __pick_nets_from_raw(
 
         return self.__ipv4_network, self.__ipv6_network
 
+    def __pick_routes_from_raw(
```

Comment:
I think I'm going to do something different here so I didn't finish this function yet.